
bool converged
bool[] fixed_cams
int16 num_cameras
int16 num_points
int16 num_observations
ObservationMsg[] observations
geometry_msgs/Point[] points
Pose3D[] cameras
int16[] points_ID
int16[] keyframes_ID
